etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -31,27 +64,6 @@
       notify:
         - Restart wazuh
 
-    - name: Check if custom certificates exists
```

Comment:
When redeploying Wazuh at Habrok, @jackhodgkiss said this:

> Also Will's patch does away with custom certificates which is required for replacing the self-signed certs with Wouter's on the dashboard. Unfortunately we cannot use them earlier as the indexer fails I think due to the root-ca being used to validate them which is fine for self-signed but not public CA certs.

@jovial did you see that issue?

Reply:
Hmm, no, but it may because I'm not doing the same thing. My method is to override `dashboard_node_name:`:

```
dashboard_node_name: "wazuh-dashboard"
```

Then I add: 

- etc/kayobe/environments/test/wazuh/certificates/wazuh-certificates/wazuh-dashboard-key.pem
- etc/kayobe/environments/test/wazuh/certificates/wazuh-certificates/wazuh-dashboard.pem

That way I only override the certificate used for the dashboard and all the internal communication uses the generated set.

Reply:
@jackhodgkiss can you try that approach in habrok and if it works let's get it into the docs.

Reply:
I've updated the docs to use this method

Reply:
This approach is much better. Before the name of the dashboard certificate matched certificates used by the indexer which if you were using external certificates for the dashboard it would fail as the CA obviously doesn't match.

Thanks for investigating @jovial